src/js/blacklist.js

```diff
@@ -0,0 +1,7 @@
+export const blackList = [
+  {
+  	name : "Chrome",
+  	version : "46.0.2490.80",
+  	os: "Linux i686"
+  }
```

Comment:
we blacklist only Chrome version 60 on Mac, also that we shouldn't blacklist the exact version, we should match `60.0`